components/scripts/lib/gradle.sh

```diff
@@ -64,6 +64,11 @@ invoke_gradle() {
     die "ERROR: The experiment cannot continue because of a non-recoverable failure while running the build: $(cat "${EXP_DIR}/errors.txt")"
   fi
 
+  if is_build_scan_metadata_missing "$run_num"; then
+    print_bl
+    die "ERROR: The experiment cannot continue because a Build Scan was not published."
+  fi
```

Comment:
Is this only relevant in online mode? (I'm asking since we are referencing a method in the build-scan-online.sh script)

What happens in offline mode?

Reply:
Nice catch. Addressed in 537856e9c6d20a2bbdf276c53ed4bd545667e9a9.